File: src/main/java/gargoyle/ct/convert/impl/FloatConverter.java
package gargoyle.ct.convert.impl;

import gargoyle.ct.convert.Converter;
import org.jetbrains.annotations.NotNull;

public class FloatConverter implements Converter<Float> {
    @Override
    public String format(Float data) {
        return String.valueOf(data);
    }

    @NotNull
    @Override
    public Float parse(@NotNull String data) {
        return Float.valueOf(data);
    }
}

File: src/main/kotlin/gargoyle/ct/convert/impl/FloatConverter.kt
package gargoyle.ct.convert.impl

import gargoyle.ct.convert.Converter

class FloatConverter : Converter<Float> {
    override fun format(data: Float): String {
        return data.toString()
    }

    override fun parse(data: String): Float {
        return data.toFloat()
    }
}
